### DISCUSSION TITLE
Continuation of issue #1551

### DISCUSSION THREAD

**Discussion by tungstengmd** [nvim-neorg/neorg#1552] [Q&A] [answered]
my norg files for some reason aren't detected

**Comment by tungstengmd** [comment]
but the help norg is

**Reply by max397574** [reply]
help file sets the filetype itself
https://github.com/nvim-neorg/neorg/blob/e5e797e6eddcb6efb1d2c3fc2612b31ad9a76cef/doc/neorg.norg#L363

**Reply by tungstengmd** [reply]
please for the love of god if this works

**Reply by tungstengmd** [reply]
so basically i need to just make an autocmd that detects if the file ext is norg and set filetype to norg.

**Reply by tungstengmd** [reply, marked as answer]
done! `autocmd BufEnter *.norg set ft=norg` should work :3

**Reply by max397574** [reply]
well yes that fixes things
but that is builtin to core actually
so something is messed up really bad for you

https://github.com/neovim/neovim/blob/11a6f3c9301b3deb71f7e5886fce3718420355be/runtime/lua/vim/filetype.lua#L827

**Reply by tungstengmd** [reply]
i dont doubt that something is indeed severely fucked but it works now so im happy

**Reply by tungstengmd** [reply]
ok so basically i removed markview cuz somehow that interfered with norg and fucked things up

**Reply by max397574** [reply]
if markview messes up norg filetype detection you should definitely open an issue there

**Reply by tungstengmd** [reply]
did that

**Reply by max397574** [reply]
perhaps reconsider the title and some wording

**Reply by max397574** [reply]
did you find the source of the issue? since it apparently wasn't markview
